libs/remix-ui/workspace/src/lib/remix-ui-workspace.tsx

```diff
@@ -531,6 +535,40 @@ export function Workspace () {
         <span className="pl-3">{'Restore'}</span>
       </div>
     </OverlayTrigger>,
+    <Dropdown.Divider className="border mt-0 mb-0 remixui_menuhr" style={{ pointerEvents: 'none' }}/>,
+    <OverlayTrigger
+    placement="right-start"
+    overlay={
+      <Tooltip id="soliditygithubactionTooltip" className="text-nowrap">
+        <span>Add Soldity Unit testing github action to the current workspace</span>
+      </Tooltip>
+    }
+  >
+    <div
+      data-id='soliditygithubaction'
+      onClick={(e) => {
+        e.stopPropagation()
+        addGithubAction()
+        _paq.push(['trackEvent', 'fileExplorer', 'workspaceMenu', 'addSolidityTesting'])
+        hideIconsMenu(!showIconsMenu)
+      }}
+    >
+      <span
+        hidden={currentWorkspace === LOCALHOST}
+        id='soliditygithubaction'
+        data-id='soliditygithubaction'
+        onClick={(e) => {
+          e.stopPropagation()
+          addGithubAction()
+          _paq.push(['trackEvent', 'fileExplorer', 'workspaceMenu', 'addSolidityTesting'])
+          hideIconsMenu(!showIconsMenu)
+        }}
+        className='fab fa-github pl-2'
+      >
+      </span>
+      <span className="pl-3">{'Add Soldity Unit test github action'}</span>
```

Comment:
Can we find a shorter title? It takes up so much space on the UI.

Reply:
Also, i'm not sure if this option should be only visible for git workspaces. What do you think?

Reply:
I would make it visible for all the workspace I think.. it gives a bit more visibilty to this feature an encourage to commit workspace to github..